### PR TITLE
ENH: Update the patch prediction example notebook based on new API

### PR DESCRIPTION
**Pull request by mostafajahanifar**
As the API has changed in the recently merged `CNNPatchPredictor` class, I have update the related example notebook to cover the changes and add some explanation about the new `merge_predictions` and `overlay_patch_prediction` functions.

### CODE REVIEW

**Review by DavidBAEpstein — COMMENTED** [review]
[GuideToReviewFiles.docx](https://github.com/TIA-Lab/tiatoolbox/files/7136590/GuideToReviewFiles.docx)

[David_review_05.docx](https://github.com/TIA-Lab/tiatoolbox/files/7136593/David_review_05.docx)

[05.ipynb.txt](https://github.com/TIA-Lab/tiatoolbox/files/7136597/05.ipynb.txt)

[diff_output.txt](https://github.com/TIA-Lab/tiatoolbox/files/7136603/diff_output.txt)

File: .travis.yml [hunk 1]
```diff
@@ -23,10 +23,8 @@ install:
 
 # Command to run tests, e.g. python setup.py test
 script:
-    - set -e # Exit on first failure (errexit)
```
Comment (shaneahmed) [inline .travis.yml:26]: @mostafajahanifar Are you sure about this change?
I think @John-P fixed this by doing flake8 test first. Please can you check this or we should use the develop version of .travis.yml

File: setup.py [hunk 2]
```diff
@@ -20,12 +20,9 @@
...
     "pandas",
     "glymur",
-    "scikit-learn>=0.23.2",
-    "scikit-image>=0.17",
-    "torchvision==0.9.1",
-    "torch==1.8.1",
+    "torch==1.8.2+cu102",
+    "torchvision==0.9.2+cu102",
     "tqdm==4.60.0",
-    "requests",
```
Comment (shaneahmed) [inline setup.py:28]: why requests is removed?

Comment (mostafajahanifar) [reply setup.py:28]: Honestly, I don't know! I didn't touch the setup.py file. However, I will include them manually in the next commit.

**Review by shaneahmed — APPROVED** [review]

**Pull request merged**